apps/backend/src/libs/types/repository.type.ts

```diff
@@ -1,5 +1,5 @@
 type Repository<T = unknown> = {
-	create(payload: unknown): Promise<T>;
+	create(payload: unknown): Promise<T | null>;
```

Comment:
```suggestion
	create(payload: unknown): Promise<T>;
```
it should always returns T